gui/resources.xml

```diff
@@ -995,27 +1028,31 @@ Vocabulary
           <property name="icon-name">mypaint-document-revert-symbolic</property>
           <signal name="activate" handler="revert_current_scratchpad_cb"/>
         </object>
+        <accelerator key="r" modifiers="GDK_CONTROL_MASK | GDK_MOD1_MASK"/>
       </child>
       <child>
         <object class="GtkAction" id="ScratchSaveAsDefault">
           <property name="label" translatable="yes" context="Menu→Scratchpad (labels), Accel Editor (labels)">Save Scratchpad as Default</property>
           <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Save the scratchpad as the default for ‘New Scratchpad’.</property>
           <signal name="activate" handler="save_scratchpad_as_default_cb"/>
         </object>
+        <accelerator key="d" modifiers="GDK_CONTROL_MASK | GDK_SHIFT_MASK | GDK_MOD1_MASK"/>
```

Comment:
:-1: I don't think we should have any shortcuts for these secondary functions of the scratchpad.